drivers/ina3221/include/ina3221_params.h

```diff
@@ -0,0 +1,142 @@
+#define INA3221_PARAM_PIN_CRT           (GPIO_UNDEF)        /**< Critical alert pin */
+#endif
+
+#ifndef INA3221_PARAM_PIN_PV
+#define INA3221_PARAM_PIN_PV            (GPIO_UNDEF)        /**< Power valid alert pin */
+#endif
+
+#ifndef INA3221_PARAM_PIN_TC
+#define INA3221_PARAM_PIN_TC            (GPIO_UNDEF)        /**< Timing control alert pin */
+#endif
+
+#ifndef INA3221_PARAM_INT_PU_PIN_WRN
+#define INA3221_PARAM_INT_PU_PIN_WRN    (0)                 /**< Enable or disable internal pull up resistor for pin WRN */
+#endif
+
+#ifndef INA3221_PARAM_INT_PU_PIN_CRT
+#define INA3221_PARAM_INT_PU_PIN_CRT    (0)                 /**< Enable or disable internal pull up resistor for pin CRT */
+#endif
+
+#ifndef INA3221_PARAM_INT_PU_PIN_PV
+#define INA3221_PARAM_INT_PU_PIN_PV     (0)                 /**< Enable or disable internal pull up resistor for pin PV */
+#endif
+
+#ifndef INA3221_PARAM_INT_PU_PIN_TC
+#define INA3221_PARAM_INT_PU_PIN_TC     (0)                 /**< Enable or disable internal pull up resistor for pin TC */
+#endif
+
+#ifndef INA3221_PARAM_CONFIG
+#define INA3221_PARAM_CONFIG    (          \
+        INA3221_ENABLE_CH1 |               \
+        INA3221_ENABLE_CH2 |               \
+        INA3221_ENABLE_CH3 |               \
+        INA3221_NUM_SAMPLES_4 |            \
+        INA3221_CONV_TIME_BADC_4156US |    \
+        INA3221_CONV_TIME_SADC_4156US |    \
+        INA3221_MODE_CONTINUOUS_SHUNT_BUS  \
+        )                                                   /**< Configuration register value */
+#endif
+
+#ifndef INA3221_PARAM_RSHUNT_MOHM_CH1
+#define INA3221_PARAM_RSHUNT_MOHM_CH1   (100)               /**< Channel 1 shunt resistance */
+#endif
+
+#ifndef INA3221_PARAM_RSHUNT_MOHM_CH2
+#define INA3221_PARAM_RSHUNT_MOHM_CH2   (100)               /**< Channel 2 shunt resistance */
+#endif
+
+#ifndef INA3221_PARAM_RSHUNT_MOHM_CH3
+#define INA3221_PARAM_RSHUNT_MOHM_CH3   (100)               /**< Channel 3 shunt resistance */
+#endif
+
+#ifndef INA3221_PARAMS
+#define INA3221_PARAMS  {                                              \
+        .i2c = INA3221_PARAM_I2C,       \
+        .addr = INA3221_PARAM_ADDR,      \
+        .pin_warn = INA3221_PARAM_PIN_WRN,   \
+        .pin_crit = INA3221_PARAM_PIN_CRT,   \
+        .pin_tc = INA3221_PARAM_PIN_TC,    \
+        .pin_pv = INA3221_PARAM_PIN_PV,    \
+        .gpio_config = (INA3221_PARAM_INT_PU_PIN_WRN << INA3221_ALERT_WRN) | \
+                       (INA3221_PARAM_INT_PU_PIN_CRT << INA3221_ALERT_CRT) | \
+                       (INA3221_PARAM_INT_PU_PIN_TC << INA3221_ALERT_TC)   | \
+                       (INA3221_PARAM_INT_PU_PIN_PV << INA3221_ALERT_PV),    \
+        .config = INA3221_PARAM_CONFIG,    \
+        .rshunt_mohm = {                        \
+            INA3221_PARAM_RSHUNT_MOHM_CH1,         \
+            INA3221_PARAM_RSHUNT_MOHM_CH2,         \
+            INA3221_PARAM_RSHUNT_MOHM_CH3          \
+        }                                          \
```

Comment:
`uncrustify` messed up the alignment of the back slashes here. (The tool cannot be configured to get this right.) Can you fix this manually? They should be placed on a column name that is a multiple of 4; unlike other indents which should be 4*n+1.